airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -103,10 +103,10 @@ public static void attemptSQLCreateAndDropTableOperations(final String outputSch
       sqlOps.dropTableIfExists(database, outputSchema, outputTableName);
     } catch (final SQLException e) {
       if (Objects.isNull(e.getCause()) || !(e.getCause() instanceof SQLException)) {
-        throw new ConnectionErrorException(e.getSQLState(), e.getErrorCode(), e.getMessage(), e);
+        throw new ConfigErrorException(e.getSQLState(), e.getErrorCode(), e.getMessage(), e);
```

Comment:
This should be 

```java
ConfigErrorException(DisplayErrorMessage(e.getSQLState(), g.getErrorCode(), e.getMessage(), e))
```

right because `ConfigErrorException` only takes in two arguments max?

Same thing for #L109